frontend/src/pages/projects/screens/detail/storage/StorageList.tsx

```diff
@@ -22,19 +29,19 @@ const StorageList: React.FC = () => {
             Add storage
           </Button>,
         ]}
-        isLoading={false}
-        isEmpty
-        loadError={undefined}
+        isLoading={!loaded}
+        isEmpty={pvcs.length === 0}
+        loadError={loadError}
         emptyState={
           <EmptyDetailsList
             title="No storage"
             description="Choose existing, or add new on cluster storage."
           />
         }
       >
-        No content
+        <StorageTable pvcs={pvcs} />
       </DetailsSection>
-      <AddStorageModal isOpen={isOpen} onClose={() => setOpen(false)} />
+      <AddStorageModal isOpen={isOpen} onClose={() => setOpen(false)} forceRefresh={forceRefresh} />
```

Comment:
Do it yourself `onClose` -- I don't think you need to bog down `AddStorageModal` with knowing about the `forceRefresh`, we could use this modal in other spots where updating the data is immaterial.